### ISSUE TITLE
the model cannot converge 

### ISSUE THREAD

**Issue by Sissuire** [creotiv/hdrnet-pytorch#29]
I came across the problem that the model can not converge..... the training loss is getting bigger and the psnr is decreased every epoch. Any idea about this situation?

**Comment by creotiv** [comment]
This problem persisted for a few years already, many questionable things
were changed but still result the same. I compare it with originals i
thinking more the 100 times, and still doesnt see much difference.
You can try compare block by block with original tensorflow network to find
the problem block, cause that the only thing that we didnt make.

Kindly yours,
Andrey Nikishaev

Areas         ML/DS/CV/Soft Dev/BizDev/Growth Hacking/Customer Rel/IT
LinkedIn      http://ua.linkedin.com/in/creotiv
GitHub        http://github.com/creotiv
Slideshare  https://www.slideshare.net/anikishaev/
AnimalHelp https://patreon.com/uah
Telegram     @anikishaev
Mobile        +380632410666
